universal_robot/bd1fce5/bd1fce5.bug

```diff
@@ -58,4 +58,4 @@ fault-codes:
   - PROGRAMMING:CONTROL-FLOW
 failure-codes:
   - SOFTWARE:LIVENESS
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Other labels might also be applicable here (in addition). The related issues report testing the (fixed) behaviour with automated scripts, and `UNRESPONSIVE` does not fit there. Maybe add `LOSS-OF-FUNCTIONALITY` or `BEHAVING-INCORRECTLY`, not sure which. @ipa-hsd @ChrisTimperley

Reply:
Following our discussion: Loss of functionality and Unresponsive both seem appropriate.